src/wallet/wallettool.cpp

```diff
@@ -147,7 +126,18 @@ bool ExecuteWalletToolFunc(const std::string& command, const std::string& name)
             WalletShowInfo(wallet_instance.get());
             wallet_instance->Flush(true);
         } else if (command == "salvage") {
-            return SalvageWallet(path);
+            bilingual_str error;
+            std::vector<bilingual_str> warnings;
+            bool ret = RecoverDatabaseFile(path, error, warnings);
+            if (!ret) {
+                for (const auto warning : warnings) {
```

Comment:
Are there any followups queued for this file, or did you want to fix this now?

```bash
wallet/wallettool.cpp:133:33: warning: loop variable 'warning' of type 'const bilingual_str' creates a copy from type 'const bilingual_str' [-Wrange-loop-analysis]
                for (const auto warning : warnings) {
                                ^
wallet/wallettool.cpp:133:22: note: use reference type 'const bilingual_str &' to prevent copying
                for (const auto warning : warnings) {
                     ^~~~~~~~~~~~~~~~~~~~
1 warning generated.
```